config.yaml

```diff
@@ -0,0 +1,43 @@
+facility: sandbox
+log_level: debug
+namespace: default
+store_kind: fleetdb
+endpoints:
+  conditionorc:
+    authenticate: true
+    oidc_audience_url:
+    oidc_client_id:
+    oidc_issuer_url:
+    oidc_scopes:
+    url: http://conditionorc-api:9001
+  fleetdb:
+    authenticate: true
+    oidc_audience_url:
+    oidc_client_id:
+    oidc_issuer_url:
+    oidc_scopes:
+    url: http://fleetdb:8000
+  nats:
+    app_name: flipflop
+    connection_timeout: 60s
+    consumer:
+      ack_wait: 5m
+      max_ack_pending: 10
+      pull: true
+      queue_group: flipflop
```

Comment:
Does this map to the consumer name in NATS?

Reply:
Ill add consumer name. Over site.

Reply:
I'm also wondering what this is actually for?

Reply:
Give devs an idea on how they should look for flipflop. I know with the nats section specifically we will be doing away with most of the nats configuration settings and it will be baked into Ctrl.